apps/console/src/features/connections/constants/connection-constants.ts

```diff
@@ -196,4 +210,20 @@ export class ConnectionManagementConstants {
         SAML: "enterprise-saml-idp",
         SWE: "swe-idp"
     };
+
+    /**
+     * Set of Connection setup guide links.
+     */
+    public static readonly DOC_LINK_DICTIONARY: Map<string, string> = new Map<string, string>()
```

Comment:
Shouldn't we use the documentation.ts extension config file for declaring the doc links?